google/cloud/pubsub/benchmarks/endurance.cc

```diff
@@ -440,7 +453,7 @@ void PublisherTask(Config const& config, ExperimentFlowControl& flow_control,
       publisher = make_publisher();
     }
     auto message = flow_control.GenerateMessage(task);
-    auto const shutdown = absl::StartsWith(message.data(), "shutdown:");
+    auto const shutdown = StartsWith(message.data(), "shutdown:");
```

Comment:
We could use a different way to signal "shutdown the benchmark", for example, using a message attribute instead of a prefix in the message data.

Reply:
Perhaps, but I'll leave that to more than a cleanup.